packages/helpers/src/customPropTypes.js

```diff
@@ -70,3 +71,9 @@ InputValidTypes.ALL = Object.values(InputValidTypes);
 
 export const RefOf = (propType = PropTypes.instanceOf(Element)) =>
   PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: propType })]);
+
+export function isMomentObjectOrNull(date) {
+  if (date === null) return true;
+
+  return momentPropTypes.momentObj(date);
```

Comment:
Any advantage here over using `moment.isMoment`? (and thus not needing a new dependency)

Reply:
Hmmm, I will check. But actually this code just moved from DatePicker helpers. But I agree it will be good not just move code, but to rethink it.

Reply:
`momentPropTypes.momentObj` will check if it is a valid moment object or not, `moment.isMoment` doesn't

Reply:
but maybe it's not necessary

Reply:
I used it because `react-dates` used it and we're using react-dates 😂